packages/documentation/copy/en/declaration-files/Publishing.md

```diff
@@ -33,7 +33,7 @@ For example:
 
 Note that the `"typings"` field is synonymous with `types`, and could be used as well.
 
-Also note that if your main declaration file is named `index.d.ts` and lives at the root of the package (next to `index.js`) you do not need to mark the `types` property, though it is advisable to do so.
+Note:  TypeScript will infer the root `*.d.ts` based on the value in `main`, which means in cases like above you do not need to mark the `types` property, though it is advisable to do so.
```

Comment:
Your change is correct, but I think we should just remove this note. It’s kind of silly to go out of our way to say that something is unnecessary, but “advisable,” without even explaining why it’s advisable. We want people to do this purely because it’s the thing that makes npmjs.com show the little TS icon on typed packages. It’s fine if people don’t even know they can skip it.

```suggestion
```